src/poliastro/core/util.py

```diff
@@ -1,32 +1,6 @@
 import numpy as np
 from numba import njit as jit
 from numpy import cos, sin
-from numpy.linalg import norm as norm
-
-
-@jit
-def eccentricity_vector(k, r, v):
-    return ((v.dot(v) - k / (norm(r))) * r - r.dot(v) * v) / k
-
-
-@jit
-def circular_velocity(k, a):
```

Comment:
Which is the reason behind shipping this to `poliatro.core.elements`?

Reply:
I'm using that module for small Astrodynamics formulas, I think it fits there better. Probably we should find a better module name for the other functions in the future.